HealthcareSystem/GraphicalEditor/Repository/MapObjectRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GraphicalEditor.Models;
+using GraphicalEditor.Repository.Intefrace;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Repository
+{
+    public class MapObjectRepository : IMapObjectRepository
+    {
+        protected IEnumerable<MapObject> objects;
+        protected IRepository fileRepository;
+        protected MapObjectRepository() {
+            objects = fileRepository.LoadMap();
+        }
+
+        public MapObjectRepository(string path)
+        {
+        }
+
+        public bool Update(MapObject mapObject)
+        {
+
+            fileRepository = new FileRepository("test.json");
+            IEnumerable<MapObject> objects = fileRepository.LoadMap();
+            var ob = objects.FirstOrDefault(e => e.MapObjectEntity.Id == mapObject.MapObjectEntity.Id);
+            if (ob != null)
+            {
+                List<MapObject> savingList = new List<MapObject>();
```

Comment:
This is not a good name for variable. 
  - avoid using the type of variable in her name
  - give her a more descriptive name

### Algorithm for better naming

- Describe the thing in a sentence or two (easier to do in Serbian, then translate).
- Remove any words that are not crucial to convey the meaning.
- Remove any words that can be inferred from the Type, containing function, containing class, or containing module.
- See if what remains sufficiently describes the thing.
- Rename.